pkg/daemon/ceph/osd/daemon.go

```diff
@@ -55,10 +55,11 @@ func StartOSD(context *clusterd.Context, osdType, osdID, osdUUID string, pvcBack
 		}
 
 		context.Executor.ExecuteCommand(false, "", "/sbin/vgchange", "-an")
-
-		context.Executor.ExecuteCommand(false, "", "/sbin/vgchange", "-ay")
```

Comment:
#3755 is working on a change here for the specific vg for the osd being started. Let's keep this line as is, and wait for #3755 to change it.

Reply:
The vgname related change is part of this PR - #3779